### ISSUE TITLE
Ongoing Difficulties with Duplicate Tags

### ISSUE THREAD

**Issue by admagnificat** [Automattic/simplenote-android#768]
Firstly, thank you all for all of your hard work on Simplenote! Even given its bugs, I have no doubt that people all over the world are getting great benefit from it [including me -- to some extent, at least! 😄].

### Expected
 
When I assign a tag, I expect that that that tag will appear in the Tags list only once -- not twice.

### Observed

Tags will sometimes appear twice in my tag list. Even if I try to manually fix this by retagging the notes with a new tag (and deleting the original tag), the new tag will sometimes appear twice as well (seemingly after sync). This happens simultaneously across Android, the web, and the Electron app, and the notes listed for the duplicate tags are identical. (I use Simplenote in all three places, all at the same time, but mostly in my browser on the desktop.)

### Reproduced

1. I changed the email address on my account.
2. When my notes didn't sync for a minute, I thought they were gone.
3. I panicked and then exported them (from another client), and then imported them back into my account.
4. I ended up (understandably) with duplicates of everything -- both notes and tags.
5. I have deleted the duplicate notes, and retagged the original list, but there are still problems with duplicate tags.
6. It may be the case that these issues are related to what seems to be strange sync behavior with Simplenote. I love Simplenote, and think that it works very well in a lot of ways, but I don't know that I've ever used a software product with such strange syncing behavior.

Many thanks for any assistance or fixes that you can offer!

**Comment by theck13** [comment]
Thank you for the compliment!

Since the duplicated data is due to importing a copy of the existing data, that is expected behavior. Therefore, I'll close this issue.  Please, reach out to us at support@simplenoteapp.com for help.
